Makefile

```diff
@@ -2,7 +2,7 @@ PYTHON3 := $(shell which python3 2>/dev/null)
 
 PYTHON := python3
 COVERAGE := --cov=pennylane_sf --cov-report term-missing --cov-report=html:coverage_html_report
-TESTRUNNER := -m pytest tests
+TESTRUNNER := -m pytest tests --tb=short
```

Comment:
How come  ``--tb=short`` was required?

Reply:
By default pytest tracebacks are very long and hard to parse easily. This just shortens it, and will make them easier to read when on Travis.